docs/conf.py

```diff
@@ -13,25 +15,25 @@
 import os
 import sys
 import toml
-import sphinx_rtd_theme
 
-sys.path.insert(0, os.path.abspath('..'))
+sys.path.insert(0, os.path.abspath(".."))
 
 # -- Project information -----------------------------------------------------
 parsed = toml.load("../pyproject.toml")
-package_info = parsed['tool']['poetry']
-project = package_info['description']
-author = ", ".join(package_info['authors'])
-release = package_info['version']
-copyright = '2019, Q-CTRL <support@q-ctrl.com>'
+package_info = parsed["tool"]["poetry"]
+project = package_info["description"]
+author = ", ".join(package_info["authors"])
+release = package_info["version"]
+copyright = "2019, Q-CTRL <support@q-ctrl.com>"  # pylint: disable=redefined-builtin
 
 # -- HTML Variables ----------------------------------------------------------
 # Variables to insert into _templates/layout.html
 # Taken from qctrl/docs, see head.html in the repo
 html_context = {
     "var_url": "https://docs.q-ctrl.com",
     "var_title": "Open Controls Python package | Q-CTRL",
-    "var_description": "Module, class and method reference for the Q-CTRL Open Controls Python package",
+    "var_description": "Module, class and method reference forthe Q-CTRL Open Controls\
+        Python package",
```

Comment:
If I understand Python line continuation correctly then this will introduce a bunch of spaces in the string. Instead you can use implicit string concatenation (or explicit if you prefer):
```
"var_description": ("Module, class and method reference for the Q-CTRL Open Controls "
                    "Python package")
```

Also add space between "for" and "the" :)

Reply:
Nice! I didn't know the implicit string concatenation before. Thanks.